common/src/main/java/com/microsoft/identity/common/Account.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.identity.common;
+
+/**
+ * Created by shoatman on 11/15/2017.
+ */
+
+public abstract class Account {
+
+    public abstract String getUniqueIdentifier();
```

Comment:
Can you stub out some javadoc on the `abstract` classes and methods? I suspect this might make things easier once we start jointly implementing classes and may help avoid of the "what should this do?'-type questions

Reply:
Come to think of it, even around the non-abstracts would be great too - doesn't need to be the next Great American Novel or anything: just enough to help the uninitiated wrap their heads around the design :)

Reply:
👍 Will do.